src/CMakeLists.txt

```diff
@@ -7,12 +7,16 @@
 
 find_package(YARP REQUIRED COMPONENTS os dev sig math)
 find_package(ICUB REQUIRED COMPONENTS iKin)
+
+set(CMAKE_MODULE_PATH_BAK ${CMAKE_MODULE_PATH})
+find_package(Gazebo REQUIRED)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH_BAK})
+
```

Comment:
This modification is required to get around conflicts between VTK and GAZEBO in Ubuntu 2022.04.
Thanks @traversaro for the patch!